src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball returnBall() {
```

Comment:
The method `returnBall()` should be renamed to `getRandomBall()` to align with the task requirements.